api/v1beta1/disruption_webhook.go

```diff
@@ -100,6 +102,17 @@ func (r *Disruption) ValidateCreate() error {
 		return fmt.Errorf("invalid disruption name: %w", err)
 	}
 
+	if specifiedEnvironment != "" {
+		specifiedEnv := r.Spec.SpecifiedEnvironment
```

Comment:
```suggestion
		disruptionEnv := r.Spec.SpecifiedEnvironment
```

Just to make it less confusing between the 2 variables.